composer.json

```diff
@@ -54,5 +54,7 @@
     "branch-alias" : {
       "dev-master" : "1.0-dev"
     }
-  }
+  },
+  "minimum-stability": "dev",
```

Comment:
This should not be added. If it blocks for testing, it is better to use the `composer config minimum-stability dev` command. Same for the prefer stable option